src/browser/modules/Stream/CypherFrame/ErrorsView/ErrorsView.tsx

```diff
@@ -146,12 +179,20 @@ class ErrorsViewComponent extends Component<ErrorsViewProps> {
   }
 }
 
-const mapStateToProps = (state: GlobalState) => {
-  return {
-    params: getParams(state),
-    neo4jVersion: getSemanticVersion(state)
-  }
+const gqlErrorsEnabled = (state: GlobalState): boolean => {
```

Comment:
despite my other comment about not being able to capture the protocol version at exception time, I think this is quite a neat way of showing either the existing error fields, or the GQL error fields